_events/sr2022/competition.md

```diff
@@ -10,7 +10,35 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
 
 ## Schedule
 
-The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 6pm.
+The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 5pm.
```

Comment:
nit: doors open 8:45 on Sunday. I'd suggest drop the second sentence here (and maybe even the whole paragraph) now that the details are in the tables below.